completions/bun.zsh

```diff
@@ -1,350 +1,828 @@
+        '--bun[Force a script or package to use Bun.js instead of Node.js (via symlinking node)]' \
+        '--cwd[Set a specific cwd]:cwd' \
+        '-c[Load config(bunfig.toml)]: :->config' \
+        '--config[Load config(bunfig.toml)]: :->config' \
+        '--extension-order[Defaults to: .tsx,.ts,.jsx,.js,.json]:extension-order' \
+        '--jsx-factory[Changes the function called when compiling JSX elements using the classic JSX runtime]:jsx-factory' \
+        '--jsx-fragment[Changes the function called when compiling JSX fragments]:jsx-fragment' \
+        '--jsx-import-source[Declares the module specifier to be used for importing the jsx and jsxs factory functions. Default: "react"]:jsx-import-source' \
+        '--jsx-runtime["automatic" (default) or "classic"]: :->jsx-runtime' \
+        '--preload[Import a module before other modules are loaded]:preload' \
+        '-r[Import a module before other modules are loaded]:preload' \
+        '--main-fields[Main fields to lookup in package.json. Defaults to --target dependent]:main-fields' \
+        '--no-summary[Don'"'"'t print a summary]' \
+        '--version[Print version and exit]' \
+        '-v[Print version and exit]' \
+        '--revision[Print version with revision and exit]' \
+        '--tsconfig-override[Load tsconfig from path instead of cwd/tsconfig.json]:tsconfig-override' \
+        '--define[Substitute K:V while parsing, e.g. --define process.env.NODE_ENV:"development". Values are parsed as JSON.]:define' \
+        '-d[Substitute K:V while parsing, e.g. --define process.env.NODE_ENV:"development". Values are parsed as JSON.]:define' \
+        '--external[Exclude module from transpilation (can use * wildcards). ex: -e react]:external' \
+        '-e[Exclude module from transpilation (can use * wildcards). ex: -e react]:external' \
+        '--loader[Parse files with .ext:loader, e.g. --loader .js:jsx. Valid loaders: js, jsx, ts, tsx, json, toml, text, file, wasm, napi]:loader' \
+        '-l[Parse files with .ext:loader, e.g. --loader .js:jsx. Valid loaders: js, jsx, ts, tsx, json, toml, text, file, wasm, napi]:loader' \
+        '--origin[Rewrite import URLs to start with --origin. Default: ""]:origin' \
+        '-u[Rewrite import URLs to start with --origin. Default: ""]:origin' \
+        '--port[Port to serve bun'"'"'s dev server on. Default: '"'"'3000'"'"']:port' \
+        '-p[Port to serve bun'"'"'s dev server on. Default: '"'"'3000'"'"']:port' \
+        '--smol[Use less memory, but run garbage collection more often]' \
+        '--minify[Minify (experimental)]' \
+        '--minify-syntax[Minify syntax and inline data (experimental)]' \
+        '--minify-identifiers[Minify identifiers]' \
+        '--no-macros[Disable macros from being executed in the bundler, transpiler and runtime]' \
+        '--target[The intended execution environment for the bundle. "browser", "bun" or "node"]: :->target' \
+        '--inspect[Activate Bun'"'"'s Debugger]:inspect' \
```

Comment:
bun test --help displays so many parameters, but I think some of them are not being used.

![2023-09-02_22-44](https://github.com/oven-sh/bun/assets/9482395/cec3a838-8fe6-4464-a196-f0978981ec81)

Does the test runner need --inspect?